### ISSUE TITLE
Automatically listing contributors

### ISSUE THREAD

**Issue by betatim** [jupyterhub/team-compass#83]
Sparked by my rant in #82: could we make a notebook like the one in the PR which fetches all the committers and people who have "replied to" more than one issue they didn't create?

I'd love to have an easy and automatic (with false positives) way of grabbing a list of people who help out but don't get counted in commit logs.

**Comment by choldgraf** [comment]
I added a section to the jupyter activity report notebook here:

https://nbviewer.jupyter.org/github/choldgraf/jupyter-activity-snapshot/blob/master/templates/jupyter-activity-template.ipynb?flush_cache=true#List-of-all-contributors-per-organization

think we can iterate on that to grab the list of contributors over a period of time? (happy to either close this issue, or keep it open)

**Comment by choldgraf** [comment]
Is this something that we could use `github-activity` for?

E.g., if I run:

```
github-activity jupyterhub/team-compass --since 2019-11-01
```

I get:

## Contributors for this release (commentors + issue/PR authors)
([GitHub contributors page for this release](https://github.com/jupyterhub/team-compass/graphs/contributors?from=2019-11-01&to=2019-11-30&type=c))

[@arnim](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Aarnim+updated%3A2019-11-01..2019-11-30&type=Issues) | [@betatim](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Abetatim+updated%3A2019-11-01..2019-11-30&type=Issues) | [@bitnik](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Abitnik+updated%3A2019-11-01..2019-11-30&type=Issues) | [@choldgraf](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Acholdgraf+updated%3A2019-11-01..2019-11-30&type=Issues) | [@consideRatio](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3AconsideRatio+updated%3A2019-11-01..2019-11-30&type=Issues) | [@GeorgianaElena](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3AGeorgianaElena+updated%3A2019-11-01..2019-11-30&type=Issues) | [@lheagy](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Alheagy+updated%3A2019-11-01..2019-11-30&type=Issues) | [@manics](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Amanics+updated%3A2019-11-01..2019-11-30&type=Issues) | [@willingc](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3Awillingc+updated%3A2019-11-01..2019-11-30&type=Issues) | [@Zsailer](https://github.com/search?q=repo%3Ajupyterhub%2Fteam-compass+involves%3AZsailer+updated%3A2019-11-01..2019-11-30&type=Issues)

**Comment by betatim** [comment]
I like this suggestion!

Triggered by reading "commentors" and thinking "that is a weird word": should we remove the stuff in brackets in the header and just make it "Contributors for this release"?

**Comment by choldgraf** [comment]
Yeah I’d be fine with that, I put it there to be a bit more explicit about what we mean, but maybe it’s enough to just link to the github activity repo section on defining contributors?

**Comment by betatim** [comment]
I am not even sure we need to be explicit about it. We know what we are doing is right. It might be different from what other projects do but hey, their loss :)

**Comment by choldgraf** [comment]
fair enough - any chance you wanna make this a PR to `github-activity`? :-)
